lib/vtls/openssl.c

```diff
@@ -4861,13 +4922,42 @@ static ssize_t ossl_send(struct Curl_cfilter *cf,
   struct ssl_connect_data *connssl = cf->ctx;
   struct ossl_ctx *octx = (struct ossl_ctx *)connssl->backend;
 
+  size_t written;
```

Comment:
As you change type for this variable, can't it be a `ssize_t` now then since that is what is returned - to save some typecasts?